frontend/src/components/bucket/BucketConfigForm.vue

```diff
@@ -104,27 +104,39 @@ const onCancel = () => {
     >
       <TextInput
         name="bucketName"
-        label="Bucket name (what other users will see)"
+        label="Bucket name *"
+        placeholder="nr-costcontainment-dev"
```

Comment:
maybe the bucket name example should be more user-friendly to distinguish it from the examples of other fields. eg: `eg: 'My Tax Documents'`

Reply:
I like that recommendation. 'My Important Documents' might be a good one to have (less generic than tax but still good example). Want to make sure there's a mention that this must be a name that hasn't been taken by others yet in this COMS instance.